other/astyle/astylerc

```diff
@@ -9,15 +10,19 @@
 
 # Padding Options
 --pad-header
---break-blocks
 --pad-oper
+#--unpad-brackets
 --unpad-paren
 --align-pointer=name
 --align-reference=name
+#--squeeze-lines=1
+#--squeeze-ws
 
 # Formatting Options
+--add-braces
 --convert-tabs
---max-code-length=120
+--max-code-length=200
```

Comment:
What is the reason for the line length increase? (Please don't say that it's a new 4k monitor you got :D)

Reply:
Mainly that astyle is doing a really poor job at breaking argument lists, so I'd rather make this manual. I'd make it 1000 if I could, but astyle limits this to 200.

Example:

```patch
-            ck_assert_msg(get_close_nodes(subtoxes[i]->dht, dht_get_self_public_key(subtoxes[i]->dht), nodes, net_family_unspec(), true,
+            ck_assert_msg(get_close_nodes(subtoxes[i]->dht, dht_get_self_public_key(subtoxes[i]->dht), nodes, net_family_unspec(),
+                                          true,
                                           true) > 0,

-bool tox_pass_encrypt(const uint8_t plaintext[], size_t plaintext_len, const uint8_t passphrase[], size_t passphrase_len,
+bool tox_pass_encrypt(const uint8_t plaintext[], size_t plaintext_len, const uint8_t passphrase[],
+                      size_t passphrase_len,
                       uint8_t ciphertext[/*! plaintext_len + TOX_PASS_ENCRYPTION_EXTRA_LENGTH */], Tox_Err_Encryption *error);
```

Reply:
But that now open the gate for 200-wide if-statements, comments, etc., and 200 columns is a bit too wide, that's a big chunk of a 1080p screen at 9pt monospace (the whole screen is around 270 columns, give or take).